### PR TITLE
Changes based on research

### PR DESCRIPTION
**Pull request by Heydon**
Changes to the implementation based on research commissioned by the accessibility team (@greenc05). Cites the research in the **Related research** section.

### CODE REVIEW

File: src/components/tabs.md [hunk 1]
```diff
@@ -172,11 +217,21 @@ Reference implementations are intended to demonstrate **what needs to be achieve
 
 ## Related research
 
-This topic does not yet have any related research available.
+During the development of the tabs implementation, the BBC accessibility team undertook research and hosted two days of usability testing with a number of blind and partially sighted participants.
```
Comment (greenc05) [inline src/components/tabs.md]: @Heydon BBC News undertook the research... with a number of keyboard only participants (this included participants with no vision impairment as well as a range of vision impairments)

File: src/components/tabs.md [hunk 2]
```diff
@@ -172,11 +217,21 @@ Reference implementations are intended to demonstrate **what needs to be achieve
...
+
+These participants were presented with four variations of the tab interface. Version 1 appeared as a tab interface but retained the semantics and behavior of an interactive table of contents (see [Initial markup](#initial-markup)). Version 2 implemented all of the recommended semantics and behaviors of the [ARIA Authoring practices](https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel) guide, including tabs only being focusable/selectable by arrow key. The remaining versions constituted compromise positions: Version 3 implemented the tab semantics but behaved as a table of contents (moving focus to panels on tab activation). Version 4 implemented [ARIA Authoring practices tabs](https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel) except without arrow navigation (tabs remained focusable using the <kbd>Tab</kbd> key).
```
Comment (greenc05) [inline src/components/tabs.md]: These participants were presented with **three** variations of the tab interface. Version 1 implemented all of the recommended semantics and behaviours of the ARIA Authoring practices guide, including tabs only being focusable/selectable by arrow key. Version 2 constituted compromise positions implementing the tab semantics but behaved as a table of contents (moving focus to panels on tab activation). Version 3 appeared as a tab interface but retained the semantics and behaviour of an interactive table of contents (see Initial markup).

File: src/components/tabs.md [hunk 3]
```diff
@@ -172,11 +217,21 @@ Reference implementations are intended to demonstrate **what needs to be achieve
...
+
+The majority of participants were uncomfortable with same-page links and felt there were already a large number of links in BBC pages to contend with. Arrow key navigation was either not discovered by participants, was confounding, or interfered with the expected behavior of browsing by arrow key using their screen reader's virtual cursor. The concept of 'tabs' as a semantic grouping mechanism was generally popular.
```
Comment (greenc05) [inline src/components/tabs.md]: The majority of participants were uncomfortable with the same-page link pattern as this did not correctly describe the content. Arrow key navigation was either not discovered by participants, or interfered with the expected behaviour of browsing by arrow key using their screen reader's virtual cursor. The concept of 'tabs' as a semantic grouping mechanism was more popular than same page links.

Accordingly, the implementation described here follows that of version 2, and degrades to an interactive table of contents where JavaScript does not run.

File: src/components/tabs.md (hunk 1 above)
Comment (Heydon) [reply src/components/tabs.md]: @greenc05 Thanks for checking. I'll incorporate the changes shortly.

**Pull request merged**